__fixtures__/statements/select.sql

```diff
@@ -2,7 +2,7 @@ WITH regional_sales AS (
         SELECT region, SUM(amount) AS total_sales
         FROM orders
         GROUP BY region
-     ), top_regions AS (
+     ), top_regions AS NOT MATERIALIZED (
```

Comment:
is there a way to keep the old fixture in place, and make another copy with the new change?

this helps ensure we don't have breaking changes for the previous syntax.